R/opts.R

```diff
@@ -401,39 +401,43 @@ backcalc_opts <- function(prior = c("reports", "none", "infections"),
 #'
 #' @description `r lifecycle::badge("stable")`
 #' Defines a list specifying the structure of the approximate Gaussian
-#'  process. Custom settings can be supplied which override the defaults.
+#' process. Custom settings can be supplied which override the defaults.
 #'
 #' @param ls_mean Numeric, defaults to 21 days. The mean of the lognormal
-#' length scale.
+#' length scale. Not used for linear kernel.
 #'
 #' @param ls_sd Numeric, defaults to 7 days. The standard deviation of the log
 #' normal length scale. If \code{ls_sd = 0}, inverse-gamma prior on Gaussian
 #' process length scale will be used with recommended parameters
-#' \code{inv_gamma(1.499007, 0.057277 * ls_max)}.
+#' \code{inv_gamma(1.499007, 0.057277 * ls_max)}. Not used for linear kernel.
+#'
+#' @param ls_min Numeric, defaults to 0. The minimum value of the length scale.
+#' Not used for linear kernel.
 #'
 #' @param ls_max Numeric, defaults to 60. The maximum value of the length
 #' scale. Updated in [create_gp_data()] to be the length of the input data if
-#' this is smaller.
+#' this is smaller. Not used for linear kernel.
 #'
-#' @param ls_min Numeric, defaults to 0. The minimum value of the length scale.
+#' @param alpha_mean Numeric, defaults to 0. The mean of the magnitude parameter
+#' of the Gaussian process kernel. Should be approximately the expected variance
+#' of the logged Rt.
 #'
-#' @param alpha_sd Numeric, defaults to 0.05. The standard deviation of the
-#'  magnitude parameter of the Gaussian process kernel. Should be approximately
+#' @param alpha_sd Numeric, defaults to 0.01. The standard deviation of the
+#' magnitude parameter of the Gaussian process kernel. Should be approximately
 #' the expected standard deviation of the logged Rt.
 #'
 #' @param kernel Character string, the type of kernel required. Currently
-#' supporting the squared exponential kernel ("se", or "matern" with
-#' 'matern_order = Inf'), 3 over 2 oder 5 over 2 Matern kernel ("matern", with
-#' `matern_order = 3/2` (default) or `matern_order = 5/2`, respectively), or
-#' Orstein-Uhlenbeck ("ou", or "matern" with 'matern_order = 1/2'). Defaulting
-#' to the Matérn 3 over 2 kernel for a balance of smoothness and
-#' discontinuities.
+#' supporting the squared exponential kernel ("se"), periodic kernel
+#' ("periodic"), Ornstein-Uhlenbeck kernel ("ou"), and the
+#' Matern kernel ("matern").
```

Comment:
Listing them in the order they appear as options in the function.

```suggestion
#' supporting the Matern kernel ("matern"), squared exponential kernel ("se"), periodic kernel, Ornstein-Uhlenbeck #' kernel ("ou"), and the periodic kernel ("periodic").
#' 
```